### ISSUE TITLE
Config Warning: "Matches multiple schemas when only one must validate."

### ISSUE THREAD

**Issue by tomdittrich** [SchoofsKelvin/vscode-sshfs#15]
```
  "sshfs.configs": [
    {
      "root": "/",
      "host": "host",
      "port": 22,
      "username": "name",
      "agent": "pageant",
      "password": true,
      "name": "name"
    }
  ]
```
VSCode Version 1.24.0 on Win7 32bit

The config entry resulting in a VSCode Problem message: "Matches multiple schemas when only one must validate."
It's set on the second line, where the opening **{** bracket stands.
The config is working right, the message appearance is a little bit confusing.

**Comment by SchoofsKelvin** [comment]
That's because "password" and "agent" are two different authentication methods, while you're only supposed to use one.

I assume you want it to use pageant, but fall back to password-based authentication if that fails? I'll edit the schema to allow multiple authentication methods.

**Comment by tomdittrich** [comment]
That's right, thanks for your fast reply and your effort. It's up to you to change the issues status.
:beers:
